content/docs/tooling/cli/_index.md

```diff
@@ -59,6 +61,481 @@ Outputs help text
 
 ---
 
+### dellctl install
+
+Installs a Dell CSI Driver and optionally installs protocol prerequisites and validates data path connectivity.
+
+##### Available Commands
+
+```bash
+  powerstore         Install the Dell CSI Powerstore driver
+```
+{{< collapse id="dellctl-install-cli-params" title="CLI Parameters" card="false" >}}
+{{<table install-flags >}}
+| Flag               | Description | Default | Mandatory |
+|:-------------------|:------------|:--------|:-----------
+| certified                                 | The certified images from registry.connect.redhat.com are used instead of the quay.io images. | false | No |
+| config-version <string>                   | Version of Container Storage Modules to install. | {{VERSION}} | No |
+| csi-node-prefix <string>                  | The prefix for all CSI nodes provisioned by the driver. | csi-node | No |
+| csi-volume-prefix <string>                | The prefix for all CSI volumes provisioned by the driver. | csivol | No |
+| csm-authorization-proxy-hostname <string> | If deploying CSM Authorization, the hostname of the Authorization Proxy Server.  | "" | No |
+| force                                     | The existing Container Storage Module resources are deleted and then recreated.  | false | No |
+| from-file <string>                        | Path to a YAML file containing configuration details for installing the CSM.  | "" | No |
+| machineconfig                             | Configure pre-requisities based on the provided block-protocol parameter. See the relevant orchestrator and platform [installation page](../../getting-started/installation/) for which prerequisites are configured.  | false | No |
+| modules <stringArray>                     | Container Storage Modules modules to install. This is provided as a comma-separated string. Supported modules are: authorization, observability, resiliency.  | "" | No |
+| namespace <string>                        | Namespace to install into (lowercase alphanumeric, may include dashes, must start/end with alphanumeric).  | "namespace" | Yes |
+| operator-install                          | Container Storage Modules Custom Resource Definitions will be installed.  | false | No |
+| output                                    | Output from dellctl install. This dumps every generated yaml to the console. | false | No |
```

Comment:
Should we highlight that there is no install if output flag is specified?

Reply:
Updated.